engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-manage-docker-manifest-lists.html

```diff
@@ -0,0 +1,44 @@
+<span page-title ng-model="repository">{{ 'View Docker Manifest Lists for Repository:' | translate }} {{ repository.name }}</span>
+
+<div data-block="messages">
+  <div bst-alert="success" ng-hide="generationTaskId === undefined">
+    <button type="button" class="close" ng-click="clearTaskId()">&times;</button>
+    <p translate>
+      Docker metadata generation has been initiated in the background.  Click
+      <a ng-href="{{ taskUrl() }}">Here</a> to monitor the progress.
+    </p>
+  </div>
+</div>
+
+<div data-extend-template="layouts/partials/table.html">
+  <div data-block="table">
+    <table class="table table-striped table-bordered" >
+
+      <thead>
+        <tr bst-table-head>
+          <th bst-table-column><span translate>Digest</span></th>
+          <th bst-table-column><span translate>Tags</span></th>
+          <th bst-table-column><span translate>Schema Version</span></th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr bst-table-row ng-repeat="item in table.rows" >
+          <td bst-table-cell>
+            {{ item.digest }}
+          </td>
+          <td>
+            <span ng-repeat="tag in tagsForManifest(item) ">
+              <a ui-sref="docker-tag.info({tagId: tag.id})">
```

Comment:
Per our in-person chat - maybe we can comma separate or vertically orient the tags rather than putting them onto a single line so we can see and pick them out more easily.

Reply:
![newversion](https://user-images.githubusercontent.com/1069779/32298576-09b14c86-bf29-11e7-8337-68ba6aeec18e.png)

Reply:
@jturel updated to
Hopefully that 's what you'll see now

Reply:
I like it.